test/unit/fetch-test.js

```diff
@@ -38,3 +38,44 @@ test('fetch without state', function (t) {
     t.equal(error.name, 'UnauthenticatedError', 'rejects with UnauthenticatedError error')
   })
 })
+
+test('fetch with non-string path', function (t) {
+  simple.mock(internals, 'fetchProperties').resolveWith({})
+
+  fetch({
+    url: 'http://example.com',
+    account: {
+      session: {
+        id: 'abc45678'
+      }
+    }
+  }, false)
+
+  t.deepEqual(internals.fetchProperties.lastCall.arg, {
+    url: 'http://example.com/session/account',
+    bearerToken: 'abc45678',
+    path: false
+  }, 'calls fetchProperties with account url and non-string path')
+  // expected behavior??
```

Comment:
ah, I think you can leave out path entirely, so that it will be `undefined`, that will cover the else path